imports/izar_protocol_v3.leo

```diff
@@ -0,0 +1,270 @@
+program izar_protocol_v3.aleo {
+    mapping izar_keeper: bool => Keepers20;
+    mapping izar_digest: bool => scalar;
```

Comment:
Do `izar_keeper` and `izar_digest` have to be a `mapping` ?

Reply:
Yes, if we need to change the keeper.